docs/source/faq.md

```diff
@@ -61,7 +61,7 @@ Instead, GASNET can leverage UCX framework for fast end efficient implementation
 UCX framework does not provide drivers, instead it relies on the drivers provided by vendors. Currently we use: OFA VERBs, Cray's UGNI, NVIDIA CUDA.
 
 #### What is the relation between UCX and OFA Verbs or Libfabrics?
-UCX, is a middleware communication layer that relies on vendors provided user level drivers including OFA Verbs or libfabrics (or any other drivers provided by another communities or vendors) to implement high-level protocols which can be used to close functionality gaps between various vendors drivers including various libfabrics providers: coordination across various drivers, multi-rail capabilities, software based RMA, AMOs, tag-matching for transports and drivers that do not support such capabilities natively.
+UCX is a middleware communication framework that relies on device drivers including RDMA, CUDA, ROCM APIs, etc. RDMA and OS-bypass network devices typically implement device drivers using the RDMA-core Linux subsystem that is supported by the UCX framework. Support for other network abstractions can be added based on requests and contributions from the community.
```

Comment:
UCX is a middleware communication framework that relies on device drivers, e.g. RDMA, CUDA, ROCM.  RDMA and OS-bypass network devices typically implement device drivers using the RDMA-core Linux subsystem that is supported by UCX.  Support for other network abstractions can be added based on requests and contributions from the community.